docker-compose.yml

```diff
@@ -10,10 +10,30 @@ services:
     networks:
```

Comment:
Question:
I couldnt get this running and it worked for me after I removed MYSQL_USER for some reason. I dont know if I need to define all 3 (User, password and root)
```yml
    environment:
     # MYSQL_USER: ${PROFILE_MYSQL_USERNAME}
      MYSQL_PASSWORD: ${PROFILE_MYSQL_PASSWORD}
      MYSQL_ROOT_PASSWORD: ${PROFILE_MYSQL_ROOT_PASSWORD}
      MYSQL_DATABASE: ${PROFILE_DB_NAME}
```

Reply:
Security reasons, you should try to avoid using the root user as much as possible.  
If you failed to define the user in teh ``.env`` then I cna easily imagine it not working nicely

(I use the user for my application)
https://github.com/Third-Floor-CSIS/cs4337-Big-Data-Group/blob/2e9640fbee6ef0e7e7b23cf47ecb9f40e8bbf250/Microservices/Profile/src/main/resources/application.properties#L5C1-L5C53